portfolio/src/main/webapp/gallery.js

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019 Google LLC
```

Comment:
I don't think we should include this. According to the Google JS style guide, looks like licensing comments aren't required / recommended. go/js-style#file-copyright

Reply:
Hmm somehow I'm not allowed to access this go link. I found a style guide at 
https://sites.google.com/a/google.com/dfa6/development/javascript-1/JavaScript-Style-Guide
but it didn't mention this issue. Anyway, have removed all license comments.